Patterns.Behavioral.State/RoadBikeState.cs

```diff
@@ -26,6 +26,11 @@ public override void Add(ComponentType type)
             CheckState();
         }
 
+        public override bool NegotiatePothole()
+        {
+            throw new System.NotImplementedException("Bike wheel buckled!");
```

Comment:
hahahah! 😂

Reply:
Would UnsupportedException be more appropriate? 😆

Reply:
yeah, good point, I've changed it.